newt/builder/build.go

```diff
@@ -39,6 +39,7 @@ import (
 	"mynewt.apache.org/newt/newt/target"
 	"mynewt.apache.org/newt/newt/toolchain"
 	"mynewt.apache.org/newt/util"
+	"runtime"
```

Comment:
Nitpick: It would be more consistent if this `"runtime"` import were moved to the top with the rest of the system imports.